### ISSUE TITLE
Model training got stuck for several hours in the mednist_tutorial notebook

### ISSUE THREAD

**Issue by saruarlive** [Project-MONAI/MONAI#347]
**Describe the bug**

In the mednist_tutorial notebook, the program is getting stuck for several hours in the training section of the model, as shown in screenshot. All the installation steps are okay as far as I know. What could be the problem?

**Screenshots**
![image](https://user-images.githubusercontent.com/24542592/81069333-04a69300-8ee2-11ea-881a-d5f86cccd120.png)

**Environment ():**
 - Windows 10
 - Python 3.7
 - MONAI available version [e.g. git commit hash]
 - CUDA/cuDNN version 10.1/ cudnn-10.1-windows10-x64-v7.6.4.38
 - Surface book GPU 

**Additional context**

**Comment by Nic-Ma** [comment]
Hi @saruarlive ,

Thanks for your bug report.
1. Can you execute any other regular PyTorch program with GPU on your Surface book?
2. If yes, could you please change the batch_size to 1 and try again?
3. If still stuck, could you please add several print log to locate `which line` code?

Sorry I don't have a windows environment to run this notebook so far.
Thanks.

**Comment by saruarlive** [comment]
> Hi @saruarlive ,
> 
> Thanks for your bug report.
> 
> 1. Can you execute any other regular PyTorch program with GPU on your Surface book?
> 2. If yes, could you please change the batch_size to 1 and try again?
> 3. If still stuck, could you please add several print log to locate `which line` code?
> 
> Sorry I don't have a windows environment to run this notebook so far.
> Thanks.

**Comment by saruarlive** [comment]
I have solved it.
I have set up data loader as shown below and it has worked (I have single GPU)
DataLoader(train_ds, batch_size=20, shuffle=True, num_workers=0, pin_memory=True)

**Comment by Nic-Ma** [comment]
Hi @saruarlive ,

I see, this is multi-processing issue of DataLoader in windows.
Thanks.
